package.json

```diff
@@ -50,16 +51,15 @@
     "envify": "^3.4.0",
     "es6-shim": "^0.34.4",
     "eslint": "^5.16.0",
-    "eslint-config-fbjs": "^2.0.0",
-    "eslint-config-prettier": "^2.6.0",
-    "eslint-plugin-babel": "^4.1.1",
-    "eslint-plugin-flowtype": "^2.17.1",
-    "eslint-plugin-jsx-a11y": "^6.0.2",
-    "eslint-plugin-prettier": "^2.3.1",
-    "eslint-plugin-react": "^7.3.0",
-    "eslint-plugin-relay": "^0.0.8",
+    "eslint-config-fbjs": "^3.1.0",
+    "eslint-config-prettier": "^4.3.0",
+    "eslint-plugin-babel": "^5.3.0",
+    "eslint-plugin-flowtype": "^3.10.3",
+    "eslint-plugin-jsx-a11y": "^6.2.1",
+    "eslint-plugin-prettier": "^3.1.0",
+    "eslint-plugin-react": "^7.13.0",
     "fbjs-scripts": "^1.1.0",
-    "flow-bin": "^0.92.0",
+    "flow-bin": "^0.98.1",
```

Comment:
Latest one is `flow-bin@0.101.0`, can you update it? Thanks!

Reply:
https://github.com/facebook/fbjs/pull/346 to be released first in order to upgrade to a newer version than `0.98.1`. Otherwise, `yarn flow` will output errors from `node_modules/fbjs`. One of the errors:
<img width="858" alt="Screen Shot 2019-06-15 at 02 50 57" src="https://user-images.githubusercontent.com/7040242/59528184-8c755e80-8f18-11e9-8160-e8c1516bb5e9.png">